src/Control/Director.php

```diff
@@ -76,40 +80,47 @@ class Director implements TemplateGlobalProvider
     private static $alternate_base_folder;
```

Comment:
Having these separate configs makes things confusing unless we don't use this base_folder to construct a URL...